src/cascadia/TerminalSettingsModel/Theme.cpp

```diff
@@ -133,13 +104,16 @@ winrt::WUX::Media::Brush ThemeColor::Evaluate(const winrt::WUX::ResourceDictiona
     {
     case ThemeColorType::Accent:
     {
+        // As of 226xx, this object has a trivial constructor and destructor...
+        // but we can do this to avoid calling RoGetActivationFactory/RoActivateInstance
+        // every time we need a theme color.
+        static wil::object_without_destructor_on_shutdown<winrt::Windows::UI::ViewManagement::UISettings> uiSettings{};

```

Comment:
Ah, btw I meant to ask: Why's the `object_without_destructor_on_shutdown` necessary? Wouldn't just using `static` work the same way?

Reply:
(I'd like to get in the habit of tying application-lifetime values that do not need destruction to the actual application lifetime; part of that will help with shutdown crashes due to bad resource teardown)

Reply:
In this case it's not a problem since we never unload the TerminalSettings.dll dynamically, but it'll leak memory if someone else were to do it in the future... A problem for future-us?